File: app/src/main/java/br/com/brqtest/database/DatabaseHelper.java
package br.com.brqtest.database;

import android.content.Context;
import android.database.sqlite.SQLiteDatabase;
import br.com.brqtest.model.Cliente;
import br.com.brqtest.model.Endereco;
import com.j256.ormlite.android.apptools.OrmLiteSqliteOpenHelper;
import com.j256.ormlite.support.ConnectionSource;
import com.j256.ormlite.table.TableUtils;

import java.sql.SQLException;

public class DatabaseHelper extends OrmLiteSqliteOpenHelper {
    private static final int VERSION = 1;
    private static final String DB_NAME = "brqtest";

    public DatabaseHelper(Context context) {
        super(context, DB_NAME, null, VERSION);
    }

    @Override
    public void onCreate(SQLiteDatabase sqLiteDatabase, ConnectionSource connectionSource) {
        try {
            TableUtils.createTable(connectionSource,Cliente.class);
            TableUtils.createTable(connectionSource,Endereco.class);
        } catch (SQLException e) {
            e.printStackTrace();
        }
    }

    @Override
    public void onUpgrade(SQLiteDatabase sqLiteDatabase, ConnectionSource connectionSource, int oldVersion, int newVersion) {

        try {
            TableUtils.dropTable(connectionSource,Cliente.class, true);
            TableUtils.dropTable(connectionSource,Endereco.class, true);
            onCreate(sqLiteDatabase,connectionSource);
        } catch (SQLException e) {
            e.printStackTrace();
        }

    }

    @Override
    public void close() {
        super.close();

    }
}

File: app/src/main/java/br/com/brqtest/databasekotlin/DatabaseHelper.kt
package br.com.brqtest.databasekotlin

import android.content.Context
import android.database.sqlite.SQLiteDatabase
import br.com.brqtest.modelkotlin.Cliente
import br.com.brqtest.modelkotlin.Endereco
import com.j256.ormlite.android.apptools.OrmLiteSqliteOpenHelper
import com.j256.ormlite.support.ConnectionSource
import com.j256.ormlite.table.TableUtils
import java.sql.SQLException


class DatabaseHelper(context: Context) : OrmLiteSqliteOpenHelper(context, DB_NAME, null, VERSION) {

    override fun onCreate(sqLiteDatabase: SQLiteDatabase, connectionSource: ConnectionSource) {
        try {
            TableUtils.createTable(connectionSource, Cliente::class.java)
            TableUtils.createTable(connectionSource, Endereco::class.java)
        } catch (e: SQLException) {
            e.printStackTrace()
        }

    }

    override fun onUpgrade(
        sqLiteDatabase: SQLiteDatabase,
        connectionSource: ConnectionSource,
        oldVersion: Int,
        newVersion: Int
    ) {

        try {
            TableUtils.dropTable<Cliente, Any>(connectionSource, Cliente::class.java, true)
            TableUtils.dropTable<Endereco, Any>(connectionSource, Endereco::class.java, true)
            onCreate(sqLiteDatabase, connectionSource)
        } catch (e: SQLException) {
            e.printStackTrace()
        }

    }

    override fun close() {
        super.close()

    }

    companion object {
        private val VERSION = 2
        private val DB_NAME = "brqtest"
    }
}
